### PR TITLE
FATES write.configs -> parameters

### PR DESCRIPTION
**Pull request by mdietze**
## Description

Added the ability for PEcAn to modify parameters in FATES

Also cleaned up the local file hacks

Plus a small lubridate:: bug fix in n_leap_days

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) 
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: models/fates/R/write.configs.FATES.R [hunk 1]
```diff
@@ -76,7 +71,7 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
    ncdf4::nc_close(domain.nc)
    
    ## SURF
-   surf.default <- "/home/carya/FATESinput/lnd/clm2/surfdata_map/surfdata_1x1_brazil_16pfts_simyr2000_c160127.nc"
+   surf.default <- system.file("surfdata_ref.nc",package = "PEcAn.FATES")
```
Comment (serbinsh) [inline models/fates/R/write.configs.FATES.R]: Thanks looks good.

File: models/fates/R/write.configs.FATES.R [hunk 2]
```diff
@@ -209,7 +210,7 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
```
Comment (serbinsh) [inline models/fates/R/write.configs.FATES.R]: Taking out this loop for now?  Or no longer needed?

**Comment by serbinsh** [comment]
Looks good

File: models/fates/R/write.configs.FATES.R [hunk 3]
```diff
@@ -187,7 +187,8 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
      }
      
      ## Special variables used in conversions
-     leafC <- pft['leafC']/100  ## percent to proportion
+#     leafC <- pft['leafC']/100  ## percent to proportion
+     leafC <- NA
```
Comment (mdietze) [inline models/fates/R/write.configs.FATES.R]: need a default if it doesn't come out of BETY

File: models/fates/R/write.configs.FATES.R (hunk 2 above)
Comment (mdietze) [inline models/fates/R/write.configs.FATES.R]: Not sure what you mean. I haven't taken out a loop anywhere here

**Pull request merged**